assets/src/edit-story/components/library/panes/media/media3p/media3pPane.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Pane } from '../../shared';
+import paneId from './paneId';
+
+const StyledPane = styled(Pane)`
+  height: 100%;
+  padding: 0;
+  overflow: hidden;
+`;
+
+const Inner = styled.div`
+  height: 100%;
+  display: grid;
+  grid:
+    'header   ' auto
+    'infinitescroll' 1fr
+    / 1fr;
+`;
+
+function Media3pPane(props) {
+  return (
+    <StyledPane id={paneId} {...props}>
+      <Inner>{__('Media3P pane to be implemented…', 'web-stories')}</Inner>
```

Comment:
Will the Media3pPane and MediaPane be completely separate implementations, or will they both share a common 'photo tiles' widget?

Reply:
I think they will share a photo tiles widget (media gallery?). This is just a first step.